reference/forms/types/options/date_input_format_description.rst.inc

```diff
@@ -0,0 +1,8 @@
+.. versionadded:: 4.3
+
+    The ``input_format`` option was introduced in Symfony 4.3.
+
+If the ``input`` option is set to ``string``, this option specifies the format
```

Comment:
I think we need to shorten this a bit as you cannot pass anything else then a string:

```rst
The ``input`` option specifies the format of the date. It must be a valid `PHP time format`_.
```

Reply:
🤔 The `input_format` option is only useful if the `input` option is set to `string`. We can put something else in the `input` option. It seems logic to me to warn the user about this.

This is how it is done in many other places (eg : If the ``widget`` option is set to ``single_text``, this option specifies the format of the input.)

Reply:
Oh I am sorry, I misread that the first sentence is talking about the `input` option (and not the `input_format` option.